### PR TITLE
Fix: RTL websites using a LTR UI are flipping (...) [ED-12220]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/editor/elements/views/container.js [hunk 1]
```diff
@@ -628,6 +631,18 @@ const ContainerView = BaseElementView.extend( {
 			this.handleGridEmptyView();
 		}
 	},
+
+	// Todo: Remove in version 3.21.0: https://elementor.atlassian.net/browse/ED-11884.
+	// Remove together with support for physical properties inside the Mega Menu & Nested Carousel widgets.
+	getPro316VersionClass() {
+		const activeProVersion = elementorFrontend?.config?.version_pro;
+
+		if ( !! activeProVersion && activeProVersion.includes( '3.16' ) ) {
```
Comment (hein-obox) [inline assets/dev/js/editor/elements/views/container.js:640]: @KingYes Pro version referenced.

File: core/kits/documents/tabs/settings-layout.php [hunk 2]
```diff
@@ -242,6 +241,29 @@ protected function register_tab_controls() {
...
 	}
 
+	// Todo: Remove in version 3.21.0: https://elementor.atlassian.net/browse/ED-11884.
+	// Remove together with support for physical properties inside the Mega Menu & Nested Carousel widgets.
+	protected function is_pro_version_316() {
+		if ( ! Utils::has_pro() ) {
+			return false;
+		}
+
+		return strpos( ELEMENTOR_PRO_VERSION, '3.16' ) !== false;
```
Comment (hein-obox) [inline core/kits/documents/tabs/settings-layout.php:251]: @KingYes Pro version referenced.

File: includes/elements/container.php [hunk 3]
```diff
@@ -158,6 +160,29 @@ protected function get_initial_config() {
...
 	}
 
+	// Todo: Remove in version 3.21.0: https://elementor.atlassian.net/browse/ED-11884.
+	// Remove together with support for physical properties inside the Mega Menu & Nested Carousel widgets.
+	protected function is_pro_version_316() {
+		if ( ! Utils::has_pro() ) {
+			return false;
+		}
+
+		return strpos( ELEMENTOR_PRO_VERSION, '3.16' ) !== false;
```
Comment (hein-obox) [inline includes/elements/container.php:170]: @KingYes Pro version referenced.

File: assets/dev/js/editor/elements/views/container.js [hunk 4]
```diff
@@ -29,7 +29,16 @@ const ContainerView = BaseElementView.extend( {
 
 	className() {
 		const isNestedClassName = this.model.get( 'isInner' ) ? 'e-child' : 'e-parent';
-		return `${ BaseElementView.prototype.className.apply( this ) } e-con ${ isNestedClassName }`;
+
+		let containerClasses = `${ BaseElementView.prototype.className.apply( this ) } e-con ${ isNestedClassName }`;
+
+		// Todo: Remove in version 3.21.0: https://elementor.atlassian.net/browse/ED-11884.
```
Comment (TzviRabinovitch) [inline assets/dev/js/editor/elements/views/container.js:35]: why 3.21 ?

**Pull request closed without merging**